casper/src/main/scala/coop/rchain/casper/LastFinalizedBlockCalculator.scala

```diff
@@ -8,18 +8,27 @@ import coop.rchain.blockstorage.deploy.DeployStorage
 import coop.rchain.casper.CasperState.CasperStateCell
 import coop.rchain.casper.util.ProtoUtil
 import coop.rchain.catscontrib.ListContrib
+import coop.rchain.models.BlockHash
 import coop.rchain.models.BlockHash.BlockHash
 import coop.rchain.shared.Log
 
 final class LastFinalizedBlockCalculator[F[_]: Sync: Log: Concurrent: BlockStore: BlockDagStorage: SafetyOracle: DeployStorage](
     faultToleranceThreshold: Float
 ) {
+
+  def sortByHash(h1: BlockHash, h2: BlockHash): Boolean =
```

Comment:
I think it would be better to move this function into `coop.rchain.models.BlockHash`. But it is just a matter of code management. Not really a hard requirement.